features/manage-position/Create.tsx

```diff
@@ -255,16 +283,27 @@ const Create = () => {
       </Grid>
 
       <Box py={4}>
-        {tokens && collateral && gcr ? (
-          <Typography>
-            Resulting CR:{" "}
-            <span style={{ color: computedCR < gcr ? "red" : "unset" }}>
-              {pricedCR?.toFixed(4)}
+        <Typography>
+          CR of newly created tokens:{" "}
```

Comment:
Can I ask a super dumb question? What does `{" "}` do? It seems to be this weird placeholder employed throughout jsx and I've never fully understood the purpose.

Reply:
It' just a space. Could be equivalen tot [`CR of newly created tokens: ${value}`]